README.md

```diff
@@ -27,173 +26,116 @@ CockroachDB is currently in alpha. See our
 [Roadmap](https://github.com/cockroachdb/cockroach/issues/2132) and
 [Issues](https://github.com/cockroachdb/cockroach/issues) for a list of features planned or in development.
 
-## Running CockroachDB Locally
-
-### Environment Setup
-
-#### Native (read: without Docker)
-
-* set up the dev environment (see [CONTRIBUTING.md](CONTRIBUTING.md))
-* `make build`
-
-#### Using Docker
-
-Install Docker! On OSX ([official docs](https://docs.docker.com/engine/installation/mac/#from-your-shell)):
-```bash
-# install docker and docker-machine:
-$ brew install docker docker-machine
-# install VirtualBox:
-$ brew cask install virtualbox
-# create the VM (this will also start it):
-$ docker-machine create --driver virtualbox default
-# if the VM exists but isn't running, start it:
-$ docker-machine start default
-# set up the environment for the docker client:
-$ eval $(docker-machine env default)
-```
-Other operating systems will have a similar set of commands. Please check Docker's documentation for more info.
-
-Pull the CockroachDB Docker image and drop into a shell within it:
-```bash
-docker pull cockroachdb/cockroach
-docker run -p 26257:26257 -p 8080:8080 -t -i cockroachdb/cockroach shell
-# root@82cb657cdc42:/cockroach#
-```
+## Quickstart 
+
+1.  [Install Cockroach DB](https://www.cockroachlabs.com/docs/install-cockroachdb.html).
+
+2.  [Start a local cluster](https://www.cockroachlabs.com/docs/start-a-local-cluster.html) with three nodes running on different ports:
+
+    ```shell
+    $ ./cockroach start --insecure &
+    $ ./cockroach start --insecure --store=node2-data --port=26258 --http-port=8081 --join=localhost:26257 &
```

Comment:
Good point about the store directories. I'll fix that. 

As for using `cockroach` instead of `./cockroach`, it's added to your path only if you do `go get` (build from source), right? All the other more likely methods (download binary, homebrew) won't automatically add it to the path. So it seems safest to not assume anything about the path and just use `./cockroach`, no?

For one vs. three nodes, it feels to me valuable to demonstrate just how easy it is to add nodes on different ports, but I can change if you and others feel strongly about this.